locale/en/about/community.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Community Committee
+layout: about.hbs
+---
+
+# Community Committee
+
+The Community Committee (CommComm) is a top-level committee in the Node.js Foundation focused on community-facing efforts. Code commits !== the only way to contribute!
+
+The CommComm has authority over community outreach including:
```

Comment:
Please correct me if anything is wrong or missing!

Reply:
>The CommComm has authority over outward-facing community outreach efforts, including:

I think this may be a bit more concise 😄